indico/htdocs/js/indico/modules/abstracts.js

```diff
@@ -74,6 +74,9 @@
             $reviewBox.find('.js-edit-review').trigger('click');
             $('body, html').animate({scrollTop: $reviewBox.offset().top}, 'fast');
             $reviewBox.find('.i-timeline-item-box').effect('highlight', {color: Palette.highlight}, 'slow');
+        }).on('click', '.ratings-switch .i-button', function() {
+            $(this).toggleClass('open');
```

Comment:
in terms of performance it doesn't really make a difference - `$(this)` is quite fast unlike `$('selector')`